src/Jackett.Common/Definitions/matrix.yml

```
@@ -0,0 +1,180 @@
+    - {id: 49, cat: TV/SD, desc: "Sorozat/EN/SD"}
+    - {id: 28, cat: TV/HD, desc: "Sorozat/HU/HD"}
+    - {id: 7, cat: TV/SD, desc: "Sorozat/HU/SD"}
+    - {id: 9, cat: XXX, desc: "XXX"}
+    - {id: 44, cat: XXX/Other, desc: "XXX/HD"}
+    - {id: 43, cat: XXX/SD, desc: "XXX/SD"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid]
+    movie-search: [q, imdbid]
+    book-search: [q]
+    music-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 0
+    options:
+      0: Uploaded
+      1: Name
+      2: Files
+      3: Comments
+      5: DLs
+      6: Seeders
+      7: Leechers
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+  - name: info_tpp
+    type: info
+    label: Results Per Page
+    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.
+
+login:
+  path: takelogin.php
+  method: post
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+  error:
+    - selector: h2:contains("Sikertelen bejelentkezés")
+      message:
+        text: "Error during login. Its likely your username or password is incorrect."
+  test:
+    path: /
+    selector: a[href="logout.php"]
+
+search:
+  paths:
+    - path: browse.php
+  inputs:
+    $raw: "{{ range .Categories }}c{{.}}=1&{{end}}"
+    search: "{{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+    # 0 active only, 1 include dead
+    incldead: 1
+    sort: "{{ .Config.sort }}"
+    type: "{{ .Config.type }}"
+
+  rows:
+    selector: table.mainouter > tbody > tr > td > table tr:has(a[href^="download.php?id="])
+
+  fields:
+    category:
+      selector: a[href^="browse.php?cat="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: cat
+    title:
+      selector: a.torrent-link b
+    details:
+      selector: a[href^="details.php?id="]
+      attribute: href
+      filters:
+        - name: prepend
+          args: "details.php?id="
+        - name: querystring
+          args: id
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    files:
+      selector: td:nth-child(5) a
+    date:
+      selector: td:contains("Feltöltve:")
+      filters:
+        - name: regexp
+          args: "Feltöltve: *([0-9]{4}-[0-9]{2}-[0-9]{2} +[0-9]{2}:[0-9]{2}:[0-9]{2})"
+        - name: append
+          args: " +01:00" # Europe/Budapest, CET
+        - name: dateparse
+          args: "yyyy-MM-dd HH:mm:ss zzz"
+    grabs:
+      selector: td:nth-child(7) b
+      filters:
+        - name: regexp
+          args: "([0-9]+) *[xX]"
```

```suggestion
    grabs:
      selector: td:nth-child(7) b
```
the grabs, seeders and leechers processor automatically extracts just the number

---

Ah, good to know, thanks!